package.json

```
@@ -76,7 +76,8 @@
     "plugin:test:ci": "nx run-many -t test:ci --projects='tag:scope:plugin' --maxParallel=2",
     "plugin:i18n-extract": "nx run-many -t i18n-extract --projects='tag:scope:plugin'",
     "generate-apis": "yarn workspace @grafana/api-clients generate-apis",
-    "generate:api-client": "yarn workspace @grafana/api-clients generate:api-client"
+    "generate:api-client": "yarn workspace @grafana/api-clients generate:api-client",
+    "generate-openapi": "yarn workspace @grafana/openapi generate-openapi"
```

```suggestion
    "generate:openapi": "yarn workspace @grafana/openapi generate-openapi"
```
Should we standardise the generate scripts to have this format?